### PR TITLE
Bump ORT to 1.15 | chore(deps)

### PR DESCRIPTION
**Pull request by justinchuby**
This change updates ORT and ort-nightly to the latest versions and cleans up the skipped tests. It also generates opset19 so that the backend tests can run.

### CODE REVIEW

File: onnxscript/tests/function_libs/torch_lib/ops_test_data.py [hunk 1]
```diff
@@ -712,11 +663,6 @@ def _where_input_wrangler(
         "nn.functional.scaled_dot_product_attention_bool_mask",
         reason="fixme: ORT crashes on Windows, segfaults randomly on Linux",
     ),
-    xfail(
-        "nn.functional.upsample_bilinear2d",
-        reason="fixme: ORT fails with invalid model: 'INVALID_ARGUMENT : Failed to load model with error: vector::_M_range_check: __n (which is 1) >= this->size() (which is 1)'",
-        test_class_name="TestOutputConsistencyFullGraph",
-    ),
     xfail(
         "nn.functional.upsample_nearest2d",
```
Comment (justinchuby) [inline onnxscript/tests/function_libs/torch_lib/ops_test_data.py:667]: upsample_bilinear2d is fixed but upsample_nearest2d is not. Weird

**Review by BowenBao — COMMENTED** [review]
LG overall, why do you need opset19?

**Comment by justinchuby** [comment]
> LG overall, why do you need opset19?
> 
> Bumping ORT introduced opset19 tests into CI?

Yes it looks like they were discovered. I didn’t look too closely at what happened

**Review by BowenBao — APPROVED** [review]

**Comment by BowenBao** [comment]
Strange, but won't block on this. Could you follow up?

**Comment by BowenBao** [comment]
Actually wait... do we want to release with Opset19? cc @abock

**Comment by justinchuby** [comment]
Sure https://github.com/microsoft/onnxscript/issues/776

**Comment by abock** [comment]
We should probably not include Opset 19 in this PR? I think while you were away @justinchuby I added support for skipping opsets in `opgen`:

```
python -m opgen -x 19 -x 20
```

**Comment by abock** [comment]
In fact, yes, let's split Opset 19 effort out of this PR since I really need ORT 1.15 for #773.

**Review by abock — CHANGES_REQUESTED** [review]
Re-run opgen to drop Opset 19 and open a separate PR for that.

```
python -m opgen -x 19 -x 20
```

**Comment by justinchuby** [comment]
Without opset19 the backend tests fail. I can dig a little deeper

**Review by abock — APPROVED** [review]
After speaking with Justin, I had misunderstood why Opset 19 was coming in. We should take it as part of the bump.

**Pull request merged**